app/src/main/java/com/kickstarter/KSApplication.java

```diff
@@ -86,7 +85,7 @@ public boolean isInUnitTests() {
   }
 
   private void checkForCrashes() {
-    final String appId = this.build.isExternal()
+    final String appId = Build.isExternal()
```

Comment:
this was crashing on app start since it's invoked in `onCreate()` for release builds and the injected `build` was `null` before this check